vuetifyx/filter_test.go

```diff
@@ -84,6 +92,73 @@ var setByQueryCases = []struct {
 		expectedSQLConds: "extract(epoch from created_at) >= ? AND extract(epoch from created_at) < ?",
 		expectedSQLArgs:  []interface{}{mustParseDatetimePickerValue("2019-08-07 00:00"), mustParseDatetimePickerValue("2019-08-09 00:00")},
 	},
+	{
+		name: "date_range_has_two_params",
+		data: FilterData([]*FilterItem{
+			{
+				Key:          "created",
+				ItemType:     ItemTypeDateRange,
+				SQLCondition: "created_at %s ?",
+			},
+		}),
+		qs: "created.lte=2019-04-11&created.gte=2019-04-10",
+		expected: FilterData([]*FilterItem{
+			{
+				Key:       "created",
+				ItemType:  ItemTypeDateRange,
+				Modifier:  ModifierBetween,
+				Selected:  true,
+				ValueFrom: "2019-04-10",
+				ValueTo:   "2019-04-11",
+			},
+		}),
+		expectedSQLConds: "created_at >= ? AND created_at <= ?",
+		expectedSQLArgs:  []interface{}{mustParseDatePickerValue("2019-04-10"), mustParseDatePickerValue("2019-04-11")},
```

Comment:
`created.lte=2019-04-11&created.gte=2019-04-10` should results `created_at >= 2019-04-10 00:00 AND created_at < 2019-04-12 00:00`